pyTigerGraph/gds/gsql/dataloaders/nodepiece_loader.gsql

```diff
@@ -87,12 +88,20 @@ CREATE QUERY nodepiece_loader_{QUERYSUFFIX}(
                 POST-ACCUM s.@tmp_id = getvid(s);
         END;
         FOREACH i IN RANGE [1, max_distance] DO
-          ancs = SELECT t
-                 FROM ancs:s -(e_types:e)-v_types:t
+          LOG(TRUE, "ANCHOR MESSAGE DISTANCE", i);
```

Comment:
Is this logging kept intentionally? No preference here. just checking

Reply:
I figured it couldn't hurt.